src/Core/Content/Product/SalesChannel/Review/ProductReviewSaveRoute.php

```diff
@@ -68,13 +68,53 @@ public function getDecorated(): AbstractProductReviewSaveRoute
      * @Since("6.3.2.0")
      * @OA\Post(
      *      path="/product/{productId}/review",
-     *      summary="",
+     *      summary="Save a review for a product. Reviews have to be activated in the settings.",
```

Comment:
```
summary="Save a product review"
description="Saves a review for a product. Reviews have to be activated in the settings."
```

Reply:
summary - answer: What do I do as a user?
description - answer: What does the endpoint do?